packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2088,6 +2108,16 @@ class InputDecoration {
   /// of the [Text] widget used to display the error.
   final int errorMaxLines;
 
+  /// Whether the placeholder text floats to become a label on focus.
```

Comment:
This is called "label" in the implementation and the docs.

Reply:
Done.